templates/projects.html

```diff
@@ -74,16 +94,6 @@ <h2>Complete your organization profile</h2>
 {% endif %}
 {% endcomment %}
 
-<div style="float: right; margin: 0 0 1em 5em;">
-  <a id="new-system" href="/store?protocol=govready.com/apps/compliance/2018/information-technology-system" class="btn btn-success"><i class="glyphicon glyphicon-plus"></i> Add system </a>
-  <a id="new-project" href="/store" class="btn btn-success"><i class="glyphicon glyphicon-plus"></i> Add other app </a>
-  {% if is_lonely_admin %}
-  <a href="#" onclick="show_invite_to_org(); return false;" class="btn btn-success"><i class="glyphicon glyphicon-send"></i> Invite colleague </a>
-  {% endif %}
-</div>
-
-<h1>Your Compliance Projects</h1>
```

Comment:
As above, every page ought to have an h1.

Reply:
Added `h1` back to `projects.html`